spec/name_tamer_spec.rb

```diff
@@ -184,9 +184,12 @@
         s: 'scout-loyalty-optimizer'
       },
       { n: 'RenÃ© Descartes', t: :person, nn: 'René Descartes', sn: 'René Descartes', s: 'rene-descartes' },
-      { n: 'John â€œJonnoâ€� Johnson', t: :person, nn: 'John “Jonno” Johnson', sn: 'John Johnson', s: 'john-johnson' },
+      { n: 'John â€œJonnoâ€ Johnson', t: :person, nn: 'John “Jonno” Johnson', sn: 'John Johnson', s: 'john-johnson' },
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.